SYCL/Plugin/interop-level-zero-interop-task-mem.cpp

```diff
@@ -0,0 +1,43 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// Test for Level Zero interop_task
+
+#include <CL/sycl.hpp>
+// clang-format off
+#include <level_zero/ze_api.h>
+#include <CL/sycl/backend/level_zero.hpp>
+// clang-format on
+
+class my_selector : public cl::sycl::device_selector {
+public:
+  int operator()(const cl::sycl::device &dev) const override {
+    return (dev.get_platform().get_backend() == cl::sycl::backend::level_zero)
+               ? 1
+               : 0;
+  }
+};
+
+int main() {
+  sycl::queue sycl_queue = sycl::queue(my_selector());
+
+  ze_context_handle_t ze_context =
+      sycl_queue.get_context().get_native<sycl::backend::level_zero>();
+  std::cout << "zeContextGetStatus = " << zeContextGetStatus(ze_context)
+            << std::endl;
+
+  auto buf = cl::sycl::buffer<uint8_t, 1>(1024);
+  sycl_queue.submit([&](cl::sycl::handler &cgh) {
```

Comment:
it is better to add try/catch wrapper to catch all possible exceptions and process them properly.
see example in https://github.com/intel/llvm-test-suite/blob/intel/SYCL/ESIMD/reduction.cpp#L50-L75